ledger/sandbox/src/main/scala/platform/sandboxnext/Cli.scala

```diff
@@ -36,6 +36,15 @@ private[sandboxnext] object Cli extends SandboxCli {
         s"Deprecated: Use the Daml Driver for PostgreSQL if you need persistence.\nThe JDBC connection URL to a Postgres database containing the username and password as well. If present, $Name will use the database to persist its data."
       )
       .action((url, config) => config.copy(jdbcUrl = Some(url)))
+
+    parser
```

Comment:
do we need this configurable?

Reply:
I am not sure. We have this option configurable in other places and it was considered as an option for performance tuning. On the other hand, I tried changing the pool size for H2 with the append-only schema and the performance didn't improve. I'm open to removing this.

Reply:
AFAIK, for the append-only schema, this parameter is not used by the parallel indexer, it's only used by the read side. The description of the property could be improved accordingly.

Reply:
@rautenrieth-da Do you think that the CLI parameter is useful at all? I'm not sure if I should keep it.

Reply:
Exposing the parameter could be useful for future performance tests. Maybe change the description to 
```
The number of connections in the database connection pool used for serving ledger API requests
```
because it's only used for the Index but not the Indexer in sandbox.

Reply:
Fixed.

Reply:
just a side not, I do not care much: but this was not helping at all, I would not add if it is not needed